nucliadb/src/nucliadb/search/search/find.py

```diff
@@ -278,4 +309,4 @@ async def query_parser_from_find_request(
         rank_fusion=rank_fusion,
         reranker=reranker,
     )
-    return query_parser
+    return (query_parser, rank_fusion, reranker)
```

Comment:
Yeah, but then the rank_fusion and reranker are optionals, because it's only used in find, so either you `type: ignore` and assume they'll be there or return them here... Ideally, the query parser for find would be different from the current and we'd get rank_fusion and reranker from it